### PR TITLE
UcrScript Redo

### LINKED ISSUE #5868: Redo from the script window
This might be a duplicate issue. 

In the script window you are allowed to undo. I find it useful to be able to redo as well. This feature can be added on the right click and also enable the shortcut to work, In Rstudio it is **(ctrl+shift+Z)** and I think in  most softwares

### PR DESCRIPTION
**Pull request by Patowhiz**
Fixes #5868 .

@africanmathsinitiative/developers this is ready for review.

Please note, just like the way the current supported undo is a simple one, this PR aims to implement a simple Redo. Of course it's possible to implement a fully undo and redo just like word or any other code editor but that's a doable long shot.

Vb.NET doesn't offer direct Redo functionality for the VB.NET, I had to develop a work around to achieve this functionality.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@shadrackkibet Please could you review this one? thanks

**Comment by shadrackkibet** [comment]
Yes. I will do it today.

File: instat/ucrScript.vb [hunk 1]
```diff
@@ -204,21 +207,52 @@ Public Class ucrScript
...
+        'Determine if last operation can be undone in text box.   
+        If txtScript.CanUndo Then
+            bIsUserTextChangeEvent = False
+            txtScript.Undo() 'Undo the last operation.
+            mnuUndo.Enabled = False
+            mnuRedo.Enabled = True
+        End If
+    End Sub
+
+    Private Sub menu_Redo(sender As Object, e As EventArgs) Handles mnuRedo.Click
```
Comment (shadrackkibet) [inline instat/ucrScript.vb:220]:
```suggestion
    Private Sub mnuRedo_Click(sender As Object, e As EventArgs) Handles mnuRedo.Click

```
Also here?

**Comment by lloyddewit** [comment]
@rdstern Please could you test this? thanks

**Comment by lloyddewit** [comment]
@rdstern Please could you review this PR? thanks

**Pull request merged**